src/analysis/seminorm.lean

```diff
@@ -213,8 +213,12 @@ ext $ λ x, real.smul_max _ _
 instance : partial_order (seminorm 𝕜 E) :=
   partial_order.lift _ fun_like.coe_injective
 
-lemma le_def (p q : seminorm 𝕜 E) : p ≤ q ↔ (p : E → ℝ) ≤ q := iff.rfl
-lemma lt_def (p q : seminorm 𝕜 E) : p < q ↔ (p : E → ℝ) < q := iff.rfl
+lemma coe_le_coe (p q : seminorm 𝕜 E) : p ≤ q ↔ (p : E → ℝ) ≤ q := iff.rfl
+lemma coe_lt_coe (p q : seminorm 𝕜 E) : p < q ↔ (p : E → ℝ) < q := iff.rfl
```

Comment:
These should be stated in the other direction, and should probably be `norm_cast` and `simp`.